depends/packages/libXext.mk

```diff
@@ -1,12 +1,12 @@
 package=libXext
-$(package)_version=1.3.2
+$(package)_version=1.3.3
 $(package)_download_path=http://xorg.freedesktop.org/releases/individual/lib/
 $(package)_file_name=$(package)-$($(package)_version).tar.bz2
-$(package)_sha256_hash=f829075bc646cdc085fa25d98d5885d83b1759ceb355933127c257e8e50432e0
+$(package)_sha256_hash=b518d4d332231f313371fdefac59e3776f4f0823bcb23cf7c7305bfb57b16e35
 $(package)_dependencies=xproto xextproto libX11 libXau
 
 define $(package)_set_vars
-  $(package)_config_opts=--disable-static
+  $(package)_config_opts=--disable-static --enable-malloc0returnsnull
```

Comment:
I looked into this a few weeks back, and I believe this is auto-detected, but we manage to get it wrong.

Could you please explain exactly why this is necessary? I'm afraid we're accidentally building against system files.

Reply:
I mentioned this a little here: https://github.com/bitcoin/bitcoin/pull/15844#issuecomment-485025030

Also, it's recommended when cross-compiling here: https://www.x.org/wiki/CrossCompilingXorg/

Reply:
You're completely right, thank you.

Could you add that link (or c/p the quote) as a comment? It's very helpful to know that it's for bypassing an AC_RUN_IFELSE.

Reply:
Actually, after looking at libXext's source I believe the stricter `--disable-malloc0returnsnull` works as well (I also confirmed thru a gitian build). I'll update this.